### ISSUE TITLE
Documentatie-verbetering: omschrijving zaaktype identificatie

### ISSUE THREAD

**Issue by sergei-maertens** [VNG-Realisatie/gemma-zaken#1584]
Op dit moment zegt de omschrijving van `identificatie` bij zaaktype:

```
Unieke identificatie van het ZAAKTYPE binnen de CATALOGUS waarin het ZAAKTYPE voorkomt.
```

Dit zou moeten zijn:

```
Unieke identificatie van het ZAAKTYPE binnen de CATALOGUS waarin het ZAAKTYPE voorkomt.

Dezelfde identificatie kan meerdere keren voorkomen binnen dezelfde catalogus, mits de zaaktypen geen overlappende geldigheid hebben. Zaaktypen met dezelfde identificatie worden dan beschouwd als verschillende versies van elkaar.
```

**Comment by michielverhoef** [comment]
@joeribekker Kun jij dit oppakken?

**Comment by MNIJ** [comment]
Dit klinkt als een heel wenselijke verbetering. Is er echter een business rule die dit nu voorschrijft? Met andere woorden, is dit gedrag al onderdeel van de specificatie? Zelf kan ik dat nergens vinden en het is daarmee niet helemaal duidelijk of een implementatie die deze wijze van zaaktype-versionering afdwingt aan de specificatie voldoet.

**Comment by michielverhoef** [comment]
Wordt meegenomen in de komende release.

**Comment by sergei-maertens** [comment]
@MNIJ 

> Dit klinkt als een heel wenselijke verbetering. Is er echter een business rule die dit nu voorschrijft? Met andere woorden, is dit gedrag al onderdeel van de specificatie? Zelf kan ik dat nergens vinden en het is daarmee niet helemaal duidelijk of een implementatie die deze wijze van zaaktype-versionering afdwingt aan de specificatie voldoet.

Ja, de referentie-implementatie geeft de doorslag bij twijfel over de specificatie/beschreven gedrag. En dit is het gedrag in de referentie-implementatie :-)
